app.js

```diff
@@ -50,4 +50,5 @@ function clearActiveClasses() {
     slides.forEach((slide) => {
         slide.classList.remove('active');
     })
-}
\ No newline at end of file
+}
+
```

Comment:
nit: extra line

Reply:
I have written the code in app.js from line 55 to 76 for the animations in the second line, which I have deleted now, so this extra line(Line 54) is no longer there. Line 53 is the last line of code in app.js now